src/app/hooks.ts

```diff
@@ -0,0 +1,5 @@
+import { useDispatch, useSelector } from "react-redux";
+import { AppDispatch, RootState } from "./store";
+
+export const useAppSelector = useSelector.withTypes<RootState>();
+export const useAppDispatch = useDispatch.withTypes<AppDispatch>();
```

Comment:
The `useDispatch` hook does not have a `withTypes` method. You should cast the `useDispatch` hook to the `AppDispatch` type like this: `export const useAppDispatch = () => useDispatch<AppDispatch>();`.